default.hbs

```diff
@@ -87,19 +87,23 @@ $(document).ready(function () {
 
     // Filter
     $('.filter-selection').click(function(event) {
-        let classToFilter = event.target.innerHTML.replace(/\s+/g, '-').toLowerCase();
-        let hashToFilter;
-        if (classToFilter[0] == '#') {
-            hashToFilter = classToFilter.slice(1);
-        }
+        let classToFilter = event.target.innerHTML;
+        let internalTag = $(this).hasClass('internal-tag');
```

Comment:
Not sure how this part works? If there's an internal tag present, at least one of the tags will be `tag-hash-something`. I don't see any `internal-tag` class.

Reply:
The `internal-tag` class is hard-coded in the filter for conversations, insights and reflections in `page-stories.hbs`, as I thought those were the only internal tags we will filter for. Or are there more I'll have to factor in?

Reply:
Oh I see now, didn't notice that... Thanks!